Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -108,14 +108,18 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Pool Actors Handlers"))
 	TArray<FPoolObjectHandle> PoolActorHandlersInternal;
 
-	/** Array of actors spawned */
-	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Spawned Actors Internal"))
-	TArray<AActor*> SpawnedStarActorsInternal;
-
 	/** Store the material for dynamic progress materil fill for a star actor */
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star Dynamic Progress Material"))
 	TObjectPtr<UMaterialInstanceDynamic> StarDynamicProgressMaterial = nullptr;
 
+	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star Material Slot Divisor"))
+	FName StarMaterialSlotDivisor = TEXT("Percentage2");
+	
+	// Temporary used to tweak the stars with bad UV  to look as expected
+	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star Material Fractional Divisor Temporarry"))
+	float StarMaterialFractionalDivisor = 3.f;
+	
+
```

Comment:
Extra space, please reformat all changed files from this PR, just in case

Reply:
i believe it's fixed: https://github.com/h4rdmol/ProgressionSystem/pull/17/commits/1667bc35491808340e600ea0cafa0b31dc34e408